From dd0b7f456689e3859ddf5f041eac59cd38f6cf84 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@foss.st.com>
Date: Tue, 13 Aug 2024 17:30:28 +0200
Subject: [PATCH 1081/1695] plat-stm32mp2: default enable CFG_STM32_GPU_OPP

Enable CFG_STM32_GPU_OPP by default.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 9d261d795d6e43177d02432bc717fd7523ca337a plat-stm32mp2: default enable CFG_STM32_GPU_OPP]
Signed-off-by: Pascal Paillet <p.paillet@foss.st.com>
Change-Id: I90e5e0b760cd3d9b9acfe4ca3c95e038c7876ac8
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/398313
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Pascal PAILLET <p.paillet@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index ce0ddd0c7..6d8335c36 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -159,6 +159,7 @@ CFG_SCMI_CORTEXM_AGENT ?= y
 CFG_STM32_BSEC3 ?= y
 CFG_STM32_BSEC_WRITE ?= y
 CFG_STM32_CPU_OPP ?= y
+CFG_STM32_GPU_OPP ?= y
 CFG_STM32_CRYP ?= y
 CFG_STM32_EXTI ?= y
 CFG_STM32_FMC ?= y
-- 
2.47.2

